confluence-xml/src/test/resources/confluencexml/include/entities.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<hibernate-generic>
+  <object class="Space" package="com.atlassian.confluence.spaces">
+    <id name="id">100</id>
+    <property name="name"><![CDATA[TestSpace]]></property>
+    <property name="key"><![CDATA[TestSpace]]></property>
+  </object>
+
+  <object class="Page" package="com.atlassian.confluence.pages">
+    <id name="id">10</id>
+    <property name="space" class="Space" package="com.atlassian.confluence.spaces">
+      <id name="id">100</id>
+    </property>
+    <property name="title"><![CDATA[TestContent]]></property>
+    <property name="parent" class="Page" package="com.atlassian.confluence.pages">
+      <id name="id">12</id>
+    </property>
+    <collection name="bodyContents">
+      <element class="BodyContent" package="com.atlassian.confluence.core">
+        <id name="id">0</id>
+      </element>
+    </collection>
+  </object>
+
+  <object class="BodyContent" package="com.atlassian.confluence.core">
+    <id name="id">0</id>
+    <property name="body">
+      <![CDATA[
+          <p>This page includes another page:</p>
+          <ac:structured-macro ac:name="include" ac:schema-version="1" data-layout="default" ac:local-id="0c4cd4d7-167b-455f-aacc-d793ea14616a" ac:macro-id="e2cca8926332c500394f85da339d3bf0">
+            <ac:parameter ac:name="">
+                <ac:link>
+                    <ri:page ri:content-title="Includee" ri:version-at-save="1" />
+                </ac:link>
+            </ac:parameter>
+          </ac:structured-macro>
+          <p>Additional includes from the JIRA ticket:</p>
+          <ac:structured-macro ac:name="include" ac:schema-version="1" data-layout="default" ac:local-id="ea6dc2a5-b3cf-4814-9b32-c1df544f5c12" ac:macro-id="b08110a09d42c6f5b956444b43a90dcd">
+              <ac:parameter ac:name="">
+                  <ac:link>
+                      <ri:page ri:content-title="recently-updated" ri:version-at-save="9" />
+                  </ac:link>
+              </ac:parameter>
+          </ac:structured-macro>
+          <ac:structured-macro ac:name="include" ac:schema-version="1" data-layout="default" ac:local-id="19e3604c-71d5-4cec-9db3-0585f69fbf50" ac:macro-id="c9e0971c9e6125325ca414a4dba375d7">
+              <ac:parameter ac:name="">
+                  <ac:link>
+                      <ri:page ri:content-title="Attachments" ri:version-at-save="3" />
+                  </ac:link>
+              </ac:parameter>
+          </ac:structured-macro>
+          <ac:structured-macro ac:name="include" ac:schema-version="1" data-layout="default" ac:local-id="50209bd0-0cda-49c2-b953-fb464fb80096" ac:macro-id="2434bd1cd37303222861cd3c97af1e30">
+              <ac:parameter ac:name="">
+                  <ac:link>
+                      <ri:page ri:content-title="note, tip, info, success, warning, error" ri:version-at-save="4" />
+                  </ac:link>
+              </ac:parameter>
+          </ac:structured-macro>
+      ]]>
```

Comment:
You should have kept the excerpt-include IMO, just showing that it's imported in a generic way (no rename of the parameter).